srcpkgs/foo2zjs/template

```diff
@@ -0,0 +1,28 @@
+# Template file for 'foo2zjs'
+pkgname=foo2zjs
+version=20171202
+revision=1
+wrksrc=foo2zjs
+build_style=gnu-makefile
+hostmakedepends="pkg-config ghostscript groff"
+makedepends="ghostscript foomatic-db-engine"
+depends="ghostscript foomatic-db-engine"
+short_desc="Collection of print drivers by rkkda.com"
+maintainer="Jasper Chan <jasperchan515@gmail.com>"
+license=GPL-2.0-or-later
+homepage="foo2zjs.rkkda.com"
+distfiles="http://foo2zjs.rkkda.com/foo2zjs.tar.gz"
```

Comment:
needs a versioned distfile

Reply:
They don't seem to offer one, I've looked again since the last time we talked about this, it hasn't changed. 

There have been updates since this pr was first opened so the checksum doesn't match anymore so I don't really know what to do about this.